### PR TITLE
feature/ADH-4624 +hottest & cached files

### PR DESCRIPTION
**Pull request by KuzminIA**
Task: https://tracker.yandex.ru/ADH-4624

### CODE REVIEW

File: smart-frontend/app/src/store/adh/cachedFiles/cachedFilesTableSlice.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+});
+
+const cachedFilesTableSlice = createTableSlice({
+  name: 'adh/cachedFilesTable',
+  createInitialState,
+  reducers: {},
+  extraReducers: () => {},
+});
+
+export const { setPaginationParams, cleanupTable, setSortParams, setFilter, resetFilter } =
```
Comment (remizov-arena) [inline smart-frontend/app/src/store/adh/cachedFiles/cachedFilesTableSlice.ts]: set alias, pls, for this methods, as `cleanupTable: cleanupCachedFilesTable` etc

File: smart-frontend/app/src/components/pages/ClusterInfoPage/ClusterInfoPage.module.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.clusterInfoPageContent {
+  display: flex;
+  flex-direction: column;
```
Comment (remizov-arena) [inline smart-frontend/app/src/components/pages/ClusterInfoPage/ClusterInfoPage.module.scss]: what are reasons for this styles ?

**Review by dbardin90 — APPROVED** [review]

File: smart-frontend/app/src/store/adh/cachedFiles/cachedFilesTableSlice.ts (hunk 1 above)
Comment (KuzminIA) [reply smart-frontend/app/src/store/adh/cachedFiles/cachedFilesTableSlice.ts]: fixed, thank you

File: smart-frontend/app/src/components/pages/ClusterInfoPage/ClusterInfoPage.module.scss (hunk 2 above)
Comment (KuzminIA) [reply smart-frontend/app/src/components/pages/ClusterInfoPage/ClusterInfoPage.module.scss]: removed

**Review by remizov-arena — APPROVED** [review]

**Pull request merged**